src/creatures/players/player.hpp

```diff
@@ -100,11 +100,8 @@ class Player final : public Creature, public Cylinder, public Bankable {
 	Player(const Player &) = delete;
```

Comment:
Maybe use `using X_ptr = std::shared_ptr<X>`, on class `X`. (`using Player_ptr = std::shared_ptr<Player>;`)
What do you say? We use it elsewhere as ProtocolX.